### DISCUSSION TITLE
Monitoring nominatim data update changes

### DISCUSSION THREAD

**Discussion by kenseii** [osm-search/Nominatim#2367] [Q&A] [answered]
Hello,
Having a nominatim server running continuous updates with `./utils/update.php --import-osmosis-all` is there a way of knowing or estimating which amount of storage is consumed every time there is an update?

Also i saw that nominatim has the `import_osmosis_log` table but i think it only records the first import only. Is it possible to log the updates there too?

**Comment by mtmail** [comment, marked as answer]
It writing only the first import into `import_osmosis_log` was a bug. Should be fixed in the latest release, see https://github.com/osm-search/Nominatim/discussions/2350

OpenStreetMap data grows irregular and unpredictable, there's too many factors to consider in how people add data to OpenStreetMap https://www.osmstats.neis-one.org/?item=changesets And it doesn't map well to number of database records or index sizes. It's better to create a separate report outside of Nominatim of the raw postgresql data directory size.

**Reply by kenseii** [reply]
Thank you, i think i will write a script that monitors the size of the folder.
